mlir/test/Dialect/Rock/invalid.mlir

```diff
@@ -38,3 +38,54 @@ func.func @gemm_c_too_big(%a: memref<2048x2048x2048xf32>,
   : memref<2048x2048x2048xf32> = memref<2048x2048x2048xf32> * memref<2048x2048x2048xf32>
   func.return
 }
+
```

Comment:
Since you are updating the unit test, could you add one that test a dimension size to go over (std::numeric_limits<int32_t>::max()) and expect that error to happen?